### PR TITLE
Add ghostbusters story

### PR DESCRIPTION
**Pull request by Ebadifar**
Adding my contribution to create-your-own-adventure

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/marshmallow.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Do you:
 
 [Give the marshmallow entity a turing test](turing-test/turing-test.md)
 
+[Yell for Egon](ghostbuster/ghostbuster.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/ghostbusters/ghostbusters.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+You begin to yell for Egon.
```
Comment (casey-collab) [inline english/ghostbusters/ghostbusters.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**